apps/trading/client-pages/liquidity/liquidity.tsx

```diff
@@ -30,11 +30,103 @@ import { Link, useParams } from 'react-router-dom';
 
 export const Liquidity = () => {
   const params = useParams();
-  const { pubKey } = useVegaWallet();
+  const marketId = params.marketId;
+  return <LiquidityViewContainer marketId={marketId} />;
+};
+
+export const LiquidityContainer = ({
```

Comment:
suggestion (non blocking): Can we move this to components if its used by multiple pages

Reply:
Or perhaps this should be in libs/liquidity?

Reply:
it is not really used by multiple pages. 
we have two different components one for trade grid and one for liquidity view. 
everything that is re-usable is in the lib.